spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -457,21 +457,21 @@ of the spec as laid out in this document.
 For completeness the proposal supports clients that wish to directly
 control LRAs and participants. To support this class of user an instance of
 <<source-LRAClient,LRA Client API>>
-can be instantiated directly or injected if the client is using CDI. This
+which can be instantiated directly or injected if the client is using CDI. This
```

Comment:
it still doesn't read correctly. If it is the word *class" that is the problem how about replacing with the equivalent "type" or "kind" or "To support such users and instance of ... can be ...", etc

Reply:
my bad, reverted only the second line